benchopt/cli/helpers.py

```diff
@@ -40,6 +44,141 @@ def clean(benchmark, token=None, filename=None):
+        else:
+            print("> no dependencies")
+        # availability in env (if relevant)
+        if env_name is not None:
+            # check for dependency avaulability
+            if cls.is_installed(env_name):
+                print(colorify(u'\u2713', GREEN), end='', flush=True)
+                print(colorify(f" available in env '{env_name}'", GREEN))
+            else:
+                print(colorify(u'\u2717', RED), end='', flush=True)
+                print(colorify(f" not available in env '{env_name}'", RED))
+
+        print("-" * 10)
+
+
+@helpers.command(
+    help="List information and requirements (solvers/datasets) "
+    "for a given benchmark.",
+    epilog="To (re-)install the required solvers and datasets "
+    "in a benchmark-dedicated conda environment or in your own "
+    "conda environment, see the command `benchopt install`."
+)
+@click.argument('benchmark', type=click.Path(exists=True),
+                autocompletion=complete_benchmarks)
+@click.option('--env', '-e', 'env_name',
+              flag_value='True', type=str, default='False',
+              help="Additional checks for requirement availability in "
+              "the dedicated conda environment for the benchmark "
+              "named 'benchopt_<BENCHMARK>'.")
+@click.option('--env-name', 'env_name',
+              metavar="<env_name>", type=str, default='False',
+              shell_complete=complete_conda_envs,
+              help="Additional checks for requirement availability in "
+              "the conda environment named <env_name>.")
+def info(benchmark, env_name):
+
+    # benchmark
+    benchmark = Benchmark(benchmark)
+    print(f"Info regarding '{benchmark.name}'")
+
+    # get solvers and datasets in the benchmark
+    solvers = benchmark.get_solvers()
+    datasets = benchmark.get_datasets()
+
+    # Get a list of all conda envs
+    default_conda_env, conda_envs = list_conda_envs()
+
+    # Check conda env (if relevant)
+
+    # If env_name is False (default), check availability
+    # in the current environement.
+    if env_name == 'False':
+        # check if any current conda environment
+        if default_conda_env is not None:
+            env_name = default_conda_env
+        else:
+            raise RuntimeError("No conda environment is activated.")
+    else:
+        # If env_name is True, the flag `--env` has been used. Check the conda
+        # env dedicated to the benchmark. Else, use the <env_name> value.
+        if env_name == 'True':
+            env_name = f"benchopt_{benchmark.name}"
+        else:
+            # check provided <env_name>
+            # (to avoid empty name like `--env-name ""`)
+            if len(env_name) == 0:
+                raise RuntimeError("Empty environment name.")
+            if env_name not in conda_envs:
+                raise RuntimeError(
+                    f"{env_name} is not an existing conda environment."
+                )
```

Comment:
Maybe we should put this in a function as it is used in multiple places no?

Reply:
Yes, I think it is also used in `benchopt.cli.main.install` (at least). Where would you recommend to put such a function ?

Altough, it is a bit different here, since I added:
```python
if default_conda_env is not None:
    env_name = default_conda_env
```
compared to `benchopt.cli.main.install` for instance, but I should be able to factorize this anyway.

Reply:
Done in 3149b41